### ISSUE TITLE
Minimum/maximum on number fields truncated (regression)

### ISSUE THREAD

**Issue by gward** [koxudaxi/datamodel-code-generator#1265]
**Describe the bug**

Starting with datamodel-codegen 0.18.0, the handling of `minimum` and `maximum` on number fields changed: they are now truncated in the generated code. This was handled fine under 0.17.2.

**To Reproduce**

Example schema:
```yaml
openapi: 3.0.3
info:
  title: Product API
  version: 1.0
paths:
  /product:
    post:
      operationId: createProduct
      description: Create new product
      requestBody:
        required: true
        content:
          application/json:
            schema:
              $ref: '#/components/schemas/product'
      responses:
        '200':
          content:
            application/json:
              schema:
                $ref: '#/components/schemas/product'
components:
  schemas:
    product:
      type: object
      properties:
        price:
          type: number
          minimum: -999999.999999
          maximum: 999999.999999
```

Used commandline:
```
$ venv/bin/datamodel-codegen --disable-timestamp --input-file-type openapi --input openapi.yml --output views.py
```

Output:
```
# generated by datamodel-codegen:
#   filename:  openapi.yml

from __future__ import annotations

from typing import Optional

from pydantic import BaseModel, confloat


class Product(BaseModel):
    price: Optional[confloat(ge=-999999.0, le=999999.0)] = None
```

**Expected behavior**

This is what datamodel-codegen 0.17.2 produces, which is exactly what I expect:

```
# generated by datamodel-codegen:
#   filename:  openapi.yml

from __future__ import annotations

from typing import Optional

from pydantic import BaseModel, confloat


class Product(BaseModel):
    price: Optional[confloat(ge=-999999.999999, le=999999.999999)] = None
```

**Version:**
 - OS: Ubuntu 22.10
 - Python version: 3.10.7
 - datamodel-code-generator version: 0.18.0

**Additional context**

Regression: output under 0.17.2 is exactly what I want. The bug is new in 0.18.0.

**Comment by gward** [comment]
I just used `git bisect` to track it down to this commit:

```
commit ba09d77c708fc6c8a1e1fbb6a32e47e3f9afe97a
Author: Koudai Aono <koxudaxi@gmail.com>
Date:   Mon Apr 10 02:57:01 2023 +0900

    Fix max min round issue (#1235)
    
    * Fix max min round issue
    
    * Change type name
    
    * Fix coverage

 .../model/pydantic/base_model.py                   | 29 ++++---
 datamodel_code_generator/parser/jsonschema.py      |  7 +-
 datamodel_code_generator/types.py                  | 22 +++++
 docs/custom_templates.md                           | 94 ++++++++++++++++++++++
 .../main_disable_appending_item_suffix/output.py   |  2 +-
 .../output.py                                      |  2 +-
 .../main/main_with_field_constraints/output.py     |  2 +-
 .../main/main_without_field_constraints/output.py  |  2 +-
 tests/data/openapi/api_constrained.yaml            |  1 +
 9 files changed, 139 insertions(+), 22 deletions(-)
 create mode 100644 docs/custom_templates.md
```

**Comment by kmorrisCAL** [comment]
Came here to report this issue as well.
